test/pouch_cli_images_test.go

```diff
@@ -38,6 +39,20 @@ func (suite *PouchImagesSuite) TearDownTest(c *check.C) {
 	// TODO add cleanup work
 }
 
+// TestImagesQuietOption is to verify the quiet flag.
+func (suite *PouchImagesSuite) TestImagesQuietFlag(c *check.C) {
+	qOut, _, err := runCmd(exec.Command("pouch", "images", "-q"))
+	c.Assert(err, check.IsNil)
+
+	quietOut, _, err := runCmd(exec.Command("pouch", "images", "--quiet"))
+	c.Assert(err, check.IsNil)
+
+	c.Assert(qOut, check.Equals, quietOut)
+	if match, _ := regexp.MatchString("^[0-9a-f]+\n$", qOut); !match {
```

Comment:
^[0-9a-f] :  this may be changed to ^[0-9] as image ID only contains number.

Reply:
Hi @Letty5411 , the ID is short for SHA256-Digest which is hexadecimal. ID can contains the a-f.